### ISSUE TITLE
Error messages in Domoticz log after udpate to Domoticz 2023.2

### ISSUE THREAD

**Issue by Git-Bruno** [homebridge/homebridge#3408]
### Current Situation

Error messages are posted in the Domoticz 2023.2 log every 5s when the Homebridge service runs.  The error messages are caused by an invalid format used for calls to the JSON API in Domoticz.
With Domoticz 2023.2 the format has changed as described at https://www.domoticz.com/wiki/Domoticz_API/JSON_URL's.
Requests for device information are done with the syntax '/json.htm?type=devices' while it should be  '/json.htm?type=command&param=getdevices' to be accepted by Domoticz 2023.2.

### Logs

```markdown
2023-07-30 09:00:50.566 [WebServer] Deprecated RType (devices) for API request. Handled via fallback (getdevices), please use correct API Command! (192.168.1.11)
```

### Configuration

```json
{
    "bridge": {
        "name": "Homebridge B7B6",
        "username": "xx:xx:xx:xx:xx:xx",
        "port": 51341,
        "pin": "xxx-xx-xxx",
        "advertiser": "avahi"
    },
    "accessories": [
        {
            "name": "Selector",
            "domoticzURL": "http://192.168.1.11",
            "domoticzPort": "8080",
            "deviceIDX": "50",
            "offValue": "0",
            "stayValue": "10",
            "nightValue": "20",
            "awayValue": "30",
            "timeout": 2000,
            "pollingInterval": 5000,
            "accessory": "DomoticzSelector"
        }
    ],
    "platforms": [
        {
            "name": "Config",
            "port": 8581,
            "auth": "form",
            "theme": "auto",
            "tempUnits": "c",
            "lang": "auto",
            "platform": "config"
        },
        {
            "name": "Domoticz",
            "server": "127.0.0.1",
            "port": "8080",
            "roomid": 0,
            "ssl": false,
            "dimFix": 0,
            "legacyBlinds": 0,
            "platform": "eDomoticz"
        },
        {
            "name": "Google Smart Home",
            "token": "xxxxxxxxx",
            "notice": "Keep your token a secret!",
            "twoFactorAuthPin": "xxxx",
            "platform": "google-smarthome"
        }
    ]
}
```

### Environment

- OS: Ubuntu Mint 5.15.0-72
- Software: Homebridge 1.6.1
- Node: 
- npm: 

### Process Supervisor

hb-service

**Comment by Git-Bruno** [comment]
This is an update to myself. The root cause was the homebridge-edomoticz  plugin. File 'homebridge-edomoticz/lib/domoticz.js' has been updated to use the API calls with the correct syntax but no update was offered in the Homebridge GUI , I guess because the version was unchanged. 
I fixed it by replacing file domoticz.js in /var/lib/homebridge/node_modules/homebridge-edomoticz  on my system by the updated one found in the homebridge-edomoticz github repo.
Additionally I made the same change to the file index.js for the homebridge-domoticz-selector-switch  plugin (no more maintained)
Hope this helps
